terraform/modules/security-groups/security_groups.tf

```diff
@@ -2,12 +2,12 @@ resource "aws_security_group" "api-backoffice" {
   name   = "api-backoffice"
   vpc_id = var.aws_vpc_id
   ingress {
-    protocol  = "-1"
-    from_port = 0
-    to_port   = 0
-    #    security_groups = [aws_security_group.api-backoffice-lb.id]
-    cidr_blocks = ["0.0.0.0/0"]
-    description = "Allow inbound access from the backoffice LB only"
+    protocol        = "tcp"
+    from_port       = 0
+    to_port         = 65535
+    security_groups = [aws_security_group.api-backoffice-lb.id]
+    cidr_blocks     = ["0.0.0.0/0"]
```

Comment:
If it's only through the lb, we shouldn't have this. I'm not sure how that works with the healthchecks.